apps/amplify-demo/package.json

```diff
@@ -3,22 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
-    "amazon-chime-sdk-component-library-react": "^2.2.0",
-    "amazon-chime-sdk-js": "^2.6.2",
```

Comment:
Already have this as a peer dependency of  `amazon-chime-sdk-component-library-react`